### PR TITLE
Add k8s targets now being used in PU

### PR DESCRIPTION
**Pull request by mabe13**
Add two generic Kubernetes make targets that have been created and are used in PU.

### CODE REVIEW

File: build/automation/lib/k8s.mk [hunk 1]
```diff
@@ -227,6 +227,19 @@ k8s-job-has-failed: ### Show whether the job failed - return: [true|""]
...
 
+k8s-run-cronjob-as-job: ### run a cronjob as a job (now) - mandatory: K8S_CRONJOB_NAME=[name], PROFILE=[name]; optional K8S_JOB_NAME=[job name];
+	# set up
+	eval "$$(make aws-assume-role-export-variables)"
+	make k8s-kubeconfig-get
+	eval "$$(make k8s-kubeconfig-export-variables)"
+	# create
+	kubectl delete job -n=$(K8S_APP_NAMESPACE) $(K8S_CRONJOB_NAME) ||:
+	kubectl create job -n $(K8S_APP_NAMESPACE) --from=cronjob/$(K8S_CRONJOB_NAME) $(K8S_CRONJOB_NAME)
+	make k8s-job K8S_JOB_NAMESPACE=$(K8S_APP_NAMESPACE) K8S_JOB_NAME=$(K8S_CRONJOB_NAME)
```
Comment (stefaniuk) [inline build/automation/lib/k8s.mk:238]: Do we need the `K8S_JOB_NAME` argument here?

File: build/automation/lib/k8s.mk [hunk 2]
```diff
@@ -227,6 +227,19 @@ k8s-job-has-failed: ### Show whether the job failed - return: [true|""]
 		--output jsonpath='{.status.conditions[?(@.type=="Failed")].status}' \
 	| tr '[:upper:]' '[:lower:]' | tr -d '\n'
 
+k8s-run-cronjob-as-job: ### run a cronjob as a job (now) - mandatory: K8S_CRONJOB_NAME=[name], PROFILE=[name]; optional K8S_JOB_NAME=[job name];
```
Comment (stefaniuk) [inline build/automation/lib/k8s.mk:230]: Can we change target signature to `k8s-job-run-cronjob-now: ### Run a cronjob now - mandatory: NAME=[name],PROFILE=[name]`, please?

File: build/automation/lib/k8s.mk [hunk 3]
```diff
@@ -227,6 +227,19 @@ k8s-job-has-failed: ### Show whether the job failed - return: [true|""]
...
+	# set up
+	eval "$$(make aws-assume-role-export-variables)"
+	make k8s-kubeconfig-get
+	eval "$$(make k8s-kubeconfig-export-variables)"
+	# create
+	kubectl delete job -n=$(K8S_APP_NAMESPACE) $(K8S_CRONJOB_NAME) ||:
+	kubectl create job -n $(K8S_APP_NAMESPACE) --from=cronjob/$(K8S_CRONJOB_NAME) $(K8S_CRONJOB_NAME)
+	make k8s-job K8S_JOB_NAMESPACE=$(K8S_APP_NAMESPACE) K8S_JOB_NAME=$(K8S_CRONJOB_NAME)
+
+k8s-get-token: ### Gets token for k8s dashboard
```
Comment (stefaniuk) [inline build/automation/lib/k8s.mk:240]: Can we change target signature to `k8s-authentication-get-token: ### Get authentication token e.g. to be used to access k8s dashboard`, please?

File: build/automation/lib/k8s.mk [hunk 4]
```diff
@@ -227,6 +227,19 @@ k8s-job-has-failed: ### Show whether the job failed - return: [true|""]
...
+	aws eks get-token --cluster-name=live-leks-cluster | jq '.status.token' | sed 's/^\"//' | sed 's/\"$$//'
```
Comment (stefaniuk) [inline build/automation/lib/k8s.mk:241]: Can we create `aws-eks-get-token` make target and implement it by as follows `	make -s docker-run-tools ARGS="..." CMD="$(AWSCLI) eks get-token --cluster-name=$(CLUSTER_NAME)" | make -s docker-run-tools CMD="jq '.status.token'" | sed 's/^\"//' | sed 's/\"$$//'`?

**Review by stefaniuk — CHANGES_REQUESTED** [review]
Hi Matt, really useful `make` targets. Thanks for that 👍  Give me a call tomorrow and I'll walk you through my comments.

File: build/automation/lib/k8s.mk (hunk 4 above)
Comment (mabe13) [reply build/automation/lib/k8s.mk]: @stefaniuk I've pushed some code to make this change, if you could check it over when you have time that would be great

File: build/automation/lib/k8s.mk (hunk 2 above)
Comment (mabe13) [reply build/automation/lib/k8s.mk]: fixed

File: build/automation/lib/k8s.mk (hunk 1 above)
Comment (mabe13) [reply build/automation/lib/k8s.mk:245]: Yes - turns out I missed a change to the `k8s-job-get-pod-name` target. It uses the variable to identify a single pod when the job is running in a namespace with multiple pods running. 

I've put in a code change to add the missing changes

**Pull request closed without merging**